src/components/builder/comparison-value-input.tsx

```diff
@@ -0,0 +1,60 @@
+import {useFormikContext} from 'formik';
+import {ExtendedComponentSchema, Utils as FormioUtils} from 'formiojs';
+import {useContext, useEffect, useState} from 'react';
+import usePrevious from 'react-use/esm/usePrevious';
+
+import {BuilderContext} from '@/context';
+import {getRegistryEntry} from '@/registry';
+import Fallback from '@/registry/fallback';
+import {comparisonValueComponentProps} from '@/registry/types';
+
+export interface simpleConditional {
+  conditional: {
+    eq: string;
+    when: string;
+    show: null | boolean;
+  };
+}
+
+export const ComparisonValueInput: React.FC = () => {
+  const {getFormComponents} = useContext(BuilderContext);
+  const {values, setFieldValue} = useFormikContext<simpleConditional>();
+
+  const [chosenComponent, setChosenComponent] = useState<ExtendedComponentSchema | null>(null);
```

Comment:
is it really necessary to track extra state for something that can easily be computed during render? I prefer to not do that since keeping state in sync is not ideal. Instead I'd rather see something like:

```ts
  const componentKey = values.conditional?.when;
  const chosenComponent: ExtendedComponentSchema = componentKey 
    ? FormioUtils.getComponent(getFormComponents(), conditional.when, false)
    : null;

  useEffect(...);

  if (chosenComponent === null) return false;

  // rest of render logic
```